sympy/polys/domains/finitefield.py

```diff
@@ -267,6 +267,8 @@ def from_sympy(self, a):
         """Convert SymPy's Integer to SymPy's ``Integer``. """
         if a.is_Integer or int_valued(a):
             return self.dtype(self.dom.dtype(int(a)))
+        elif a.is_Rational:
+            return self.dtype(self.dom.dtype(int(a.numerator))) / self.dtype(self.dom.dtype(int(a.denominator)))
```

Comment:
I don't think that converting with `int` is necessary.

Reply:
Indeed removing `int` doesn't seem to cause any issue (in fact, `self.dom.dtype` itself seems to point to `int`).
Perhaps `int` should also be removed from the previous line 269 for consistency?

Reply:
It works because `a.numerator` is already an int. However `self.dom.dtype` might not be `int` if gmpy2 or python-flint is installed.